### PR TITLE
Update Starlit to Ready to Run!

### PR DESCRIPTION
**Pull request by Fraxinus88**
I hope this works!

### CODE REVIEW

File: ports/thestarlitescape/The Starlit Escape.sh
Comment (JanTrueno) [inline ports/thestarlitescape/The Starlit Escape.sh:1]: We dont use the JELOS fix anymore.

Comment (JanTrueno) [inline ports/thestarlitescape/The Starlit Escape.sh:1]: Do we need etc/os-release?

Comment (JanTrueno) [reply ports/thestarlitescape/The Starlit Escape.sh:1]: Ah I see why now, the script is missing the MOD function: # We source custom mod files from the portmaster folder example mod_jelos.txt which containts pipewire fixes
[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"

Comment (Fraxinus88) [reply ports/thestarlitescape/The Starlit Escape.sh:1]: Okies will update!

**Review by Cebion — APPROVED** [review]

File: ports/thestarlitescape/The Starlit Escape.sh [hunk 1]
```diff
@@ -45,7 +45,4 @@ $ESUDO chmod +x "$GAMEDIR/gmloader"
```
Comment (cdeletre) [inline ports/thestarlitescape/The Starlit Escape.sh]:
```suggestion
pm_platform_helper "$GAMEDIR/gmloader"
```

Comment (cdeletre) [reply ports/thestarlitescape/The Starlit Escape.sh]: Remove https://github.com/PortsMaster/PortMaster-New/blob/834bea5913202100a4a9d2b3563fb7d175da584d/ports/thestarlitescape/The%20Starlit%20Escape.sh#L40

Replace https://github.com/PortsMaster/PortMaster-New/blob/834bea5913202100a4a9d2b3563fb7d175da584d/ports/thestarlitescape/The%20Starlit%20Escape.sh#L24C28-L24C31 by

`> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1`


Add after https://github.com/PortsMaster/PortMaster-New/blob/834bea5913202100a4a9d2b3563fb7d175da584d/ports/thestarlitescape/The%20Starlit%20Escape.sh#L16C1-L16C22

`[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"`

**Review by cdeletre — CHANGES_REQUESTED** [review]

**Pull request merged**